http-server/configuration-prerelease.yml

```diff
@@ -0,0 +1,2 @@
+githubRepo: test
+githubApiToken: ${GITHUB_ISSUES_API_TOKEN}
```

Comment:
Note this is a syntax that will use environment variables for the configuration property. Any confiugration value that needs environment variable, or is different between prod and prerelease are defined in the YML. Any other configuration values that are constant between prod and prerelease are `static final String` constants in application configuration.

I've been debating if we should have all configuration values in YML and no `static final String` constants. I'm leaning towards the current setup so we have less to manage between the two YML files. Keeping the two files in-sync could become more of a challenge if we put all configuration here, even the configuration that is the same for all environments. On the other hand splitting configuration between YML and Java code is not quite ideal either.

IMO the biggest liability/risk is likely with configuration files that drift and we have a production file that is missing something or does not work, hence I'm leaning to keep the two configs small and restricted to just secrets that need to be env variable injected or for configuration that differs between prod and non-prod.